Directory.Build.props

```diff
@@ -63,7 +63,7 @@
 
   <PropertyGroup>
     <ImportDefaultReferences>false</ImportDefaultReferences>
-    <LangVersion>11.0</LangVersion>
+    <LangVersion>latest</LangVersion>
```

Comment:
shall we ping it to a specific version instead of `latest` like before? so that, there will not be surprise later when we bump the .net version and we can control when to bump the language version.

Reply:
in general we should always want the latest language version, and we will never get a new one unless we bump the version of dotnet SDK. I do not think this is a thing we want to control.
MGC is doing this already: https://github.com/microsoft/typespec/blob/71c221d4d91da2e0dc40d114e7455ea524dafa3a/packages/http-client-csharp/generator/Directory.Build.props#L46

Reply:
and language version update will always be incremental.